.github/workflows/release.yml

```diff
@@ -11,6 +11,7 @@ permissions:
 jobs:
   build:
     uses: ./.github/workflows/haskell.yml
+    secrets: inherit
```

Comment:
Similarly like in the build workflow:

https://github.com/geniusyield/atlas/blob/086ea1b40610b550fa1864adcbb59cf65bce44d2/.github/workflows/build.yml#L21